add ruby baggage example (#28560)
DataDog/documentation: content/en/tracing/trace_collection/custom_instrumentation/ruby/dd-api.md

You can configure the propagation of context for distributed traces by injecting and extracting headers. Read [Trace Context Propagation][6] for information.

#### Baggage

> Baggage is a hash that can be accessed through the API as shown below. Baggage is propagated by default. Manipulating [Baggage][7]:

Reviewer: ```suggestion
Baggage is a hash that can be accessed through the API and is propagated by default. See the following example to manipulate [Baggage][7]:
```
